### PR TITLE
Kieran Lacey Placement Skills Test

### PR DESCRIPTION
**Pull request by kieranLacey**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,39 +0,0 @@
...
-MAIL_PASSWORD=null
-MAIL_ENCRYPTION=null
-
-PUSHER_APP_ID=
-PUSHER_APP_KEY=
-PUSHER_APP_SECRET=
-PUSHER_APP_CLUSTER=mt1
-
-MIX_PUSHER_APP_KEY="${PUSHER_APP_KEY}"
-MIX_PUSHER_APP_CLUSTER="${PUSHER_APP_CLUSTER}"
```
Comment (stevethomas) [inline .env.example:39]: no need to delete this file... it gives anyone starting on the project a starting point for `.env`; often with some useful development config / keys.

File: app/User.php [hunk 2]
```diff
@@ -26,4 +26,8 @@ class User extends Authenticatable
     protected $hidden = [
         'password', 'remember_token',
     ];
+    
+    function post() {
```
Comment (stevethomas) [inline app/User.php:30]: you should always declare method visibility even if public. 
```suggestion
    public function post() {
```

File: database/migrations/2014_10_12_000000_create_posts_table.php [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('posts', function (Blueprint $table) {
+            $table->increments('id');
+            $table->integer('userId');
```
Comment (stevethomas) [inline database/migrations/2014_10_12_000000_create_posts_table.php:18]: convention (at least in Laravel) here would be `user_id`

File: resources/views/home.blade.php [hunk 4]
```diff
@@ -17,6 +17,24 @@
...
+                <div>
+                    <label for="name" >Name:</label>
+                    <div>
+                        <input type="text" name="name">
+                    </div>
+                </div>
+                <div>
+                    <label for="description" >Description:</label>
+                    <div>
+                        <input type="text" name="description">
```
Comment (stevethomas) [inline resources/views/home.blade.php:33]:
```suggestion
                        <textarea name="description"></textarea>
```

**Review by stevethomas — APPROVED** [review]
all round, pretty good.

**Comment by kieranLacey** [comment]
Hi Steve,

Thanks for the feedback.

That all makes sense.

- Kieran

**Pull request closed without merging**